src/sajari.php

```diff
@@ -0,0 +1,442 @@
+<?php
+
+namespace Sajari;
+
+require dirname(__FILE__) . '/vendor/autoload.php';
+require dirname(__FILE__) . '/proto/doc.php';
+require dirname(__FILE__) . '/proto/query.php';
+require dirname(__FILE__) . '/document.php';
+require dirname(__FILE__) . '/search.php';
+
+interface Opt
```

Comment:
@tbillington np. Also this is the best PHP has for "go fmt", if you haven't done so already try and set it up https://github.com/FriendsOfPHP/PHP-CS-Fixer e.g. maybe you can set it up as a pre-commit hook to auto-fmt or reject the commit if it shows warnings. Note: I tried quickly to get it working with Atom, but I don't think it's working for some reason.